x/oracle/client/cli/tx.go

```diff
@@ -0,0 +1,165 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/CosmosContracts/juno/v12/x/oracle/types"
+)
+
+// GetTxCmd returns the CLI transaction commands for the x/oracle module.
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("Transaction commands for the %s module", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		GetCmdDelegateFeedConsent(),
+		GetCmdAggregateExchangeRatePrevote(),
+		GetCmdAggregateExchangeRateVote(),
+	)
+
+	return cmd
+}
+
+// GetCmdDelegateFeedConsent creates a Cobra command to generate or
+// broadcast a transaction with a MsgDelegateFeedConsent message.
+func GetCmdDelegateFeedConsent() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delegate-feed-consent [operator] [feeder]",
```

Comment:
Bit confusing, took me a few tries. 

Should change operator (which seems like to wants your valopper address) and [feeder] to [new_feeder_address]. The docs will specify but not entirely clear here (as someone who has never run an oracle before)

Reply:
Resolved in 9413c05585d6f9cd84b3d685f599a882c3d3ec11